ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -122,15 +122,30 @@ public void shouldGetTransactionByHash() {
 
   @Test
   public void shouldDropOldestTransactionWhenLimitExceeded() {
-    final Transaction oldestTransaction = createTransaction(0);
+    final Transaction oldestTransaction =
```

Comment:
This pattern is going to show up a lot. Calls to createTransaction() produce transactions all owned by the same account. Using the same account will result in most recent nonces being purged because they are further from the lowest in the pool.